integrasjonspunkt/src/main/java/no/difi/meldingsutveksling/receipt/strategy/FiksStatusStrategy.java

```diff
@@ -21,7 +21,12 @@ public void checkStatus(Conversation conversation) {
             conversation.setPollable(false);
             conversation.setFinished(true);
         }
-        conversation.addMessageStatus(messageStatus);
+        boolean hasStatus = conversation.getMessageStatuses().stream()
+                .map(MessageStatus::getStatus)
+                .anyMatch(s -> messageStatus.getStatus().equals(s));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Replace this lambda with a method reference. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-app01.difi.local:9000/coding_rules#rule_key=squid%3AS1612)